packages/tslint-config/src/rules/blueprintClassesConstantsRule.ts

```diff
@@ -19,7 +18,7 @@ export class Rule extends Lint.Rules.AbstractRule {
         optionsDescription: "Not configurable",
         optionExamples: ["true"],
         type: "style",
-        typescriptOnly: false,
+        typescriptOnly: false
```

Comment:
i think this'll fail the lint check -- make sure you're respecting the repo's `tslint.json` (it's at the root)

Reply:
Sorry about that. My IDE had prettier running by default.. and I didn't notice that I prettified your files for no additional fee! Will fix